src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2115,42 +2240,56 @@
           short-summary: Port number on which to receive traffic.
         - name: --dest-address
           short-summary: The IP address or URI at which to receive traffic.
+    examples:
+        - name: This example checks connectivity to a destination virtual machine over port 80.
+          text: az network watcher test-connectivity -g ContosoRG --source-resource MultiTierApp0 --dest-resource Database0 --dest-port 80
 """
 
 helps['network watcher show-next-hop'] = """
     type: command
     short-summary: Get information on the 'next hop' for a VM.
-    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located.
+    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located.  For more information about show-next-hop, refer to https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-check-next-hop-cli
```

Comment:
Provide a description of what `next hop` means in this context, unless it will be clear to experts who are using this command. I assume it's a topology-related query?